src/components/Subscriptions.js

```diff
@@ -75,7 +76,14 @@ const mapStateToProps = (store, ownProps) => ({
   cookies: ownProps.cookies,
   error: store.error
 })
+const mapDispatchToProps = dispatch => {
```

Comment:
where is this code used?

Reply:
After talking, this looks liek something we need to do in order to have access to `dispatch` from Redux.